### PR TITLE
Rename 'percentage' to 'percent'

### LINKED ISSUE #24: Percentage is not
So I was trying to work out what was wrong with my code until I saw this:

``` javascript
percentage: 0.5,            // Overall percentage (between 0 to 1)
```

Percentages are between 0 and 100. That's what makes them percentages 😄 , otherwise it's just a `ratio`.

There's a non-breaking way to fix this, which is to keep the existing behaviour as undocumented and to add a new `percent` field with the correct behaviour.

### PR DESCRIPTION
**Pull request by pedrorfernandes**
This fixes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ progress(request('https://az412801.vo.msecnd.net/vhd/VMBuild_20141027/VirtualBox
 .on('progress', function (state) {
     // The state is an object that looks like this:
     // {
-    //     percentage: 0.5,            // Overall percentage (between 0 to 1)
+    //     percent: 0.5,            // Overall percent (between 0 to 1)
```
Comment (satazor) [inline README.md]: Could you correct the indent here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "request-progress",
-  "version": "2.0.1",
+  "version": "3.0.1",
```
Comment (satazor) [inline package.json]: Do not bump the version, I will do it with [release-it](https://github.com/webpro/release-it)

Comment (pedrorfernandes) [inline package.json]: Done!

**Comment by satazor** [comment]
Ty, I will publish a new release once I arrive home.

**Pull request merged**